### ISSUE TITLE
doesn't compile on Go 1.0

### ISSUE THREAD

**Issue by nergdron** [nsf/golemon#1]
Attempting to install on Go 1.0 produces the following errors:

$ go get -v github.com/nsf/golemon
github.com/nsf/golemon (download)
github.com/nsf/golemon
# github.com/nsf/golemon

../nsf/golemon/lempar.go:13: syntax error: unexpected %
../nsf/golemon/lempar.go:63: non-declaration statement outside function body
../nsf/golemon/lempar.go:64: non-declaration statement outside function body
../nsf/golemon/lempar.go:65: syntax error: unexpected )
../nsf/golemon/lempar.go:187: syntax error: unexpected %, expecting }
../nsf/golemon/lempar.go:193: syntax error: unexpected %, expecting }
../nsf/golemon/lempar.go:240: syntax error: unexpected %, expecting case or default or }
../nsf/golemon/lempar.go:380: syntax error: unexpected %
../nsf/golemon/lempar.go:426: syntax error: unexpected %, expecting }
../nsf/golemon/lempar.go:452: syntax error: unexpected %, expecting case or default or }
../nsf/golemon/lempar.go:452: too many errors

**Comment by nsf** [comment]
Yes, I don't maintain it. Sorry if you really need it, try goyacc or something.

**Comment by nergdron** [comment]
Well, for anyone else looking for a parsing library for Go, peg seems quite nice:

https://github.com/pointlander/peg

**Comment by dgryski** [comment]
The problem is that Go tries to build the template file. It should be renamed to not have a .go extension.
